base/util/src/main/java/com/netscape/cmsutil/crypto/CryptoUtil.java

```diff
@@ -3216,6 +3303,27 @@ else if (algname.equals(SignatureAlgorithm.RSAPSSSignatureWithSHA512Digest.toStr
 
         throw new NoSuchAlgorithmException(method + alg);
     }
+
+    public static  byte[] getDesParity(byte[] key) throws Exception {
+        String method = "KDF.getDesParity";
```

Comment:
You moved this method into CryptoUtil.java, so you might want to change the method string to CryptoUtil.getDesParity